bug/label.go

```diff
@@ -1,19 +1,62 @@
 package bug
 
 import (
+	"crypto/sha1"
 	"fmt"
 	"io"
 	"strings"
 
 	"github.com/MichaelMure/git-bug/util/text"
 )
 
+// RGBColor is a color type in the form red, green, blue
+type RGBColor struct {
+	red   uint8
+	green uint8
+	blue  uint8
+}
+
 type Label string
 
 func (l Label) String() string {
 	return string(l)
 }
 
+// RGBColor from a Label computed in a deterministic way
+func (l Label) RGBColor() RGBColor {
+	id := 0
+	hash := sha1.Sum([]byte(l))
```

Comment:
Can we assign the colors for the labels on some fixed domain like a specific color for a particular level ?